app/controllers/spree/api/affirm_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Spree
+  module Api
+    class AffirmController < Spree::Api::BaseController
+      def payload
```

Comment:
This controller doesn't do anything in this commit, so it probably belongs in another commit.